ansible/roles/texlive/files/install-tl.profile

```diff
@@ -0,0 +1,42 @@
+# texlive.profile written on Fri Aug  2 10:41:50 2013 UTC
+# It will NOT be updated and reflects only the
+# installation profile at installation time.
+selected_scheme scheme-custom
+TEXDIR $TEXDIR
+TEXMFLOCAL $TEXMFLOCAL 
+TEXMFSYSCONFIG $TEXMFSYSCONFIG 
+TEXMFSYSVAR $TEXMFSYSVAR 
+TEXMFVAR $TEXMFVAR 
```

Comment:
Convert this file to a template: http://docs.ansible.com/ansible/template_module.html
and you can replace the above with something like

```
TEXDIR {{ texlive_dest }}/{{ texlive_version }}
TEXMFLOCAL {{ texlive_dest }}/texmf-local
...
```

E.g.
- https://github.com/manics/infrastructure/tree/docker/ansible/roles/docker/templates
- https://github.com/manics/infrastructure/blob/docker/ansible/roles/docker/tasks/main.yml#L20